scripts/expert/PortalGun.zs

```diff
@@ -7,7 +7,7 @@ print("Initializing 'PortalGun.zs'...");
 
 #Miniature Black Hole
 recipes.remove(<portalgun:item_miniature_black_hole>);
-recipes.addShaped(<portalgun:item_miniature_black_hole>, [[<ore:dustEnderPearl>, <ore:dustEnderPearl>, <ore:dustEnderPearl>], [<ore:dustEnderPearl>, <ic2:nuclear:10>, <ore:dustEnderPearl>], [<ore:dustEnderPearl>, <ore:dustEnderPearl>, <ore:dustEnderPearl>]]);
```

Comment:
Here is a small mistake from my side, because I have removed the OreDicts in the last update - thanks for the change :)